From af7e9cd1b64fea2fff360dcd8b32cf42a197d228 Mon Sep 17 00:00:00 2001
From: Robert Roth <robert.roth.off@gmail.com>
Date: Wed, 21 Aug 2013 22:48:12 +0000
Subject: Increased buffersize to be able to hold larger cpuinfo files (bgo #681632)

---
diff --git a/sysdeps/linux/sysinfo.c b/sysdeps/linux/sysinfo.c
index 9c82b7f..01821aa 100644
--- a/sysdeps/linux/sysinfo.c
+++ b/sysdeps/linux/sysinfo.c
@@ -36,7 +36,7 @@ static glibtop_sysinfo sysinfo = { .flags = 0 };
 static void
 init_sysinfo (glibtop *server)
 {
-	char buffer [16384];
+	char buffer [65536];
 	gchar ** processors;
 
 	if(G_LIKELY(sysinfo.flags)) return;
--
cgit v0.9.2
